subsys/bluetooth/host/Kconfig

```diff
@@ -1071,6 +1082,75 @@ config BT_HFP_HF
 	help
 	  This option enables Bluetooth HF support
 
+config BT_HFP_AG
+	bool "Bluetooth Handsfree profile AG Role support [EXPERIMENTAL]"
+	depends on PRINTK
+	select BT_RFCOMM
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth AG support
+
```

Comment:
All of the new options after the above should depend on `BT_HFP_AG`, so add `if BT_HFP_AG` after this and `endif` after the `BT_HFP_AG_*` options.

Reply:
updated